src/app/authorization/registration/registration.component.html

```diff
@@ -1,129 +1,236 @@
-                Password should be at least 6 and at the max 15 characters long. 
-            </div>
+          <label for="password" class="form-label">Password</label>
+          <input
+            type="password"
+            class="form-control"
+            id="pwd"
+            formControlName="pwd"
+          />
+          <div
+            class="form-text text-danger"
+            *ngIf="registrationFormValidator['pwd']?.errors?.['required'] && (registrationFormValidator['pwd']?.dirty || registrationFormValidator['pwd']?.touched)"
+          >
+            Password is required.
+          </div>
+          <div
+            class="form-text text-danger"
+            *ngIf="registrationFormValidator['pwd']?.errors?.['pattern']"
+          >
+            Password should be atleast 8 characters long and should contain one
+            number,one character and one special character
+          </div>
         </div>
-    </div>
-    <div class="row">
+      </div>
+      <div class="row">
         <div class="col-mb-3">
-            <div class="cpwd">
-            <label for="cpwd" class="form-label" style="margin-top: 10px;">Confirm Password</label>
-            <input type="password" class="form-control" id="cpwd" formControlName="cpwd">
-            <div class="form-text text-danger" *ngIf="Cpwd.errors?.['required'] && (Cpwd.dirty || Cpwd.touched)">
-                confirmnpassword password is Required..
-            </div>
-            <div class="form-text text-danger" *ngIf="Cpwd.errors?.['minlength'] || Cpwd.errors?.['maxlength'] && (Cpwd.dirty || Cpwd.touched)">
-               confirmPassword should be at least 6 and at the max 15 characters long. 
-            </div>
-            <div *ngIf="Cpwd.errors?.['confirmPasswordValidator']">
-                Passsword and Confirm Password didn't match.
-              </div>
-              <div class="form-text text-danger" *ngIf="registrationForm.errors?.['identityRevealed'] && (registrationForm.touched || registrationForm.dirty)">
-                Passsword and Confirm Password didn't match.
+          
+            <label for="cpwd" class="form-label" style="margin-top:15px"
+              >Confirm Password</label
+            >
+            <input
+              type="password"
+              class="form-control"
+              id="cpwd"
+              formControlName="cpwd"
+            />
+            <div
+              class="form-text text-danger"
+              *ngIf="registrationFormValidator['cpwd']?.errors?.['required'] && (registrationFormValidator['cpwd']?.dirty || registrationFormValidator['cpwd']?.touched)"
+            >
+              Confirmn Password is Required..
             </div>
+
+            <div
+              class="form-text text-danger"
+              *ngIf="registrationFormValidator['cpwd']?.errors?.['identityRevealed'] && registrationFormValidator['cpwd']?.touched"
```

Comment:
Please change identityRevealed to pattern

Reply:
ok i have change and done this change